src/js/search-form.js

```diff
@@ -1,3 +1,25 @@
 import app from './global/app';
 
 app.apiTest('search-form.js');
+
+const formError = document.querySelector('.FormError');
+const formInput = document.querySelector('.FormInput');
+const formSearch = document.querySelector('.FormSearch');
+
+formError.style.display = 'none';
+
+onSearchFailed();
+
+function onSearchFailed() {
+  formSearch.addEventListener('submit', event => {
```

Comment:
nie obsługuj zdarzenia w tej funkcji, to ta funkcja ma być użyta w obsłudze zdarzenia gdy żądanie nie zwróci żadnych filmów, ta funkcja ma tylko zająć się pokazywaniem komunikatu użytkownikowi